### ISSUE TITLE
Tab in KCT to show all ships you have

### ISSUE THREAD

**Issue by SoftwareGuy** [KanColleTool/KanColleTool#118]
Since I know very little QT, I thought I'd make this request.

Should be rather easy to implement. Have a tab that allows the player to see all the ships in his arsenal, optionally with their English name (should the translations be enabled) and their current stats. An "export to CSV/TXT/etc" button would also be ideal, because then I can pastebin the file and get other players to discuss fleets, tactics, etc.

The table would be contained in a Scrollable Window instead of a window-breaker (like some other apps do when they don't contain the overflow inside the scroll window).

For example:
```
[ Ship ID ] [ Ship Name ] [ HP ] [ Ammo ] [ Condition ]
[1] [ Fubuki ] [ 15 / 15 ] [ 10 / 10 ] [ 99 ]
[2] [ Hibiki ] [ 16 / 16 ] [ 10 / 10 ] [ 97 ]
[3] [ Nagato ] [ 80 / 80 ] [ 40 / 40 ] [ 9001 ]
```
And so on.

Implementation wise, we'd just have to request the port data of all the player's ships, then transfer the data into the table. Since the port API endpoint usually gets refreshed when you've shuffled ships around or new ships are born, shouldn't be "stale" for too long.

tl;dr - something like KCV's "All ships in account" window, only done better.

**Comment by Yukariin** [comment]
Wat?
![2015-07-18-102405_650x340_scrot](https://cloud.githubusercontent.com/assets/6450725/8760773/3c3b6546-2d37-11e5-8ac3-b1905d45276f.png)

**Comment by SoftwareGuy** [comment]
FML I forgot that we have that.

Closed.

**Comment by Rhextian** [comment]
what about adding sort by ship type?

**Comment by kevin01523** [comment]
hmm i guess the list lacks ship id, condition,exp, next, fuel ammo, ship type.feature wise battle prediction, enemy firepower and hp gauge.air superiority etc

**Comment by SoftwareGuy** [comment]
```feature wise battle prediction, enemy firepower and hp gauge.air superiority etc```
...That would be insane to try to implement, but having the option to export would be great. Not sure if the sorting already exists if you click the top of the columns...

**Comment by Yukariin** [comment]
> Not sure if the sorting already exists if you click the top of the columns...

See https://github.com/KanColleTool/KanColleTool/issues/118#issuecomment-122508309

**Comment by SoftwareGuy** [comment]
Yeah, I confirmed it seconds after posting that comment. I'm slowpoke.

**Comment by kevin01523** [comment]
@SoftwareGuy  '''..That would be insane to try to implement,'''

actually it already implemented on this project https://github.com/RyuuKitsune/ElectronicObserver

**Comment by Yukariin** [comment]
> actually it already implemented on this project https://github.com/RyuuKitsune/ElectronicObserver

Lol, KCV! have similar feature too. But this two tools written on C#, .Net, WPF, etc - in other words those apps is Windows only and absolutely unportable.
